docs/get.md

```diff
@@ -8,7 +8,8 @@ summary: "Get the code from GitHub and run ./Allwmake. If this fails, look into
 To build the adapter, you need to install a few dependencies and then execute the `Allwmake` script.
 
 1. Install [a compatible OpenFOAM distribution](https://precice.org/adapter-openfoam-support.html).
-2. Install [preCICE v3](https://precice.org/installation-overview.html). In case you need preCICE v2, please install an older version of the adapter (v1.2.3 is the latest release to support preCICE v2).
+2. Install [preCICE v3](https://precice.org/installation-overview.html).
+    * In case you need preCICE v2, please install an older version of the adapter (v1.2.3 was the last release to support preCICE v2).
```

Comment:
It would be good to add a link to the release archive, as finding it might not be trivial in case you don't know where to look for older releases.